release-schema.json

```
@@ -111,6 +119,16 @@
             "null"
           ],
           "minLength": 1
+        },
+        "value": {
+          "title": "Value",
+          "description": "The total upper estimated value of the framework agreement.",
+          "$ref": "#/definitions/Value"
+        },
+        "period": {
+          "title": "Period",
+          "description": "The period when the framework agreement is active.",
```

Moving in line with `tender/contractPeriod`.

```suggestion
          "description": "The period over which the framework agreement is estimated to be active.",
```